beitragsordnung.md

```diff
@@ -18,22 +21,49 @@ Die Mitgliederversammlung des Vereins Welcome Werkstatt e.V. hat am 9.11.2019 fo
     | Kinder (bis zum vollendeten 14. Lebensjahr) | 7,50 Euro | - |
     | Senioren (ab vollendetem 65. Lebensjahr) | 15 Euro | Ab 7,50 Euro |
 
-5. Für die Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Alle anderen Personen (Gäste) entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal mindestens 10 Euro.
+Der Quartale bzw. (Halb-)Jährliche Beitrag berechnet sich entsprechend.
+
+
+2. Tagessatz
+2.1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Alle anderen Personen (Gäste) entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal mindestens 10 Euro.
```

Comment:
"Mindestens 10 Euro" finde ich zu unkonkret 
```suggestion
2.1. Für die private Nutzung der Räumlichkeiten des Vereins entrichten ordentliche Mitglieder mit Stimmrecht eine reduzierte Tagespauschale von 3 Euro. Gäste entrichten pro Nutzungstag innerhalb der regulären Öffnungszeiten pauschal 10 Euro.
```

Reply:
Wir können das "mindestens" rausnehmen. Bisher stand es in der Beitragsordnung.

Reply:
Muss hier nochmal auf das Stimmrecht verwiesen werden? Das ist doch durch ordentlich schon gegeben.